lib-intro.tex

```diff
@@ -32,13 +32,13 @@
 defined.'' That is, whether the operator is a member is unspecified.
 
 \pnum
-Requirements are stated in terms of concepts~(Concepts TS [dcl.spec.concept]).
-Concepts are stated in terms of well-defined expressions that define valid terms of
-the types that satisfy the \oldoldtxt{requirements}\newnewtxt{concept}. For every set of
-well-defined expression requirements there is a \oldoldtxt{table}\newnewtxt{named concept} that
+Requirements are stated in terms of \added{concepts~(Concepts TS [dcl.spec.concept]).
+Concepts are stated in terms of} well-defined expressions that define valid terms of
+the types that satisfy the \changed{requirements}{concept}. For every set of
+well-defined expression requirements there is a \changed{table}{named concept} that
 specifies an initial set of the valid expressions and their semantics. Any generic
 algorithm (Clause~\ref{algorithms}) that uses the well-defined expression requirements
-is described in terms of the valid expressions for its template type parameters.
+is described in terms of the valid expressions for its formal type parameters. 
```

Comment:
It's "template" in the latest draft, but "formal" in 4141. Reverting to the 4141 text.